cli/collection.go

```diff
@@ -60,6 +62,24 @@ func MakeCollectionCommand(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if name != "" {
+				fetchedCols := cols
+				cols = nil
+				for _, c := range fetchedCols {
+					if c.Name() == name {
+						cols = append(cols, c)
+						break
+					}
+				}
+			}
```

Comment:
thought: This seems superfluous. If `name != ""`, there will be exactly one collection anyways.

thought (out-of-scope): We should eventually ensure that the flags that are part of the switch statement are mutually exclusive.